mqtt/handler.go

```diff
@@ -64,7 +63,8 @@ type handler struct {
 
 // NewHandler creates new Handler entity.
 func NewHandler(publishers []messaging.Publisher, es redis.EventStore,
-	logger logger.Logger, auth policies.AuthServiceClient) session.Handler {
+	logger logger.Logger, auth policies.AuthServiceClient,
+) session.Handler {
```

Comment:
Is this change necessary (because of the linter)?

Reply:
It is not necessary